travis-ci.sh

```diff
@@ -31,7 +31,11 @@ if [ $e2eCode -ne 0 ]; then
     ./sonobuoy logs
     mkdir results; tar xzf $outFile -C results
     find results
-    find results/plugins -exec cat {} \;
+    find results/plugins -type f \
+      -exec echo {} \; \
+      -exec cat {} \; \
+      -exec echo \; \
```

Comment:
And you could echo `\n` directly here using `echo -e "\n\n"`

Reply:
Seemed like 6 on one hand, half a dozen on the other.

I wasn't sure if the `-e` flag was as cross-platform supported. If you know one way or the other I'd be curious.

Reply:
Sorry, I don't know :( In that case, probably just best to leave it as is.